lazyflow/operators/classifierOperators.py

```diff
@@ -484,10 +455,20 @@ def execute(self, slot, subindex, roi, result):
             
             probabilities = full_probabilities
 
+        # Cancel out masked pixels.
+        if mask is not None:
+            mask_all_channels = numpy.broadcast_to(mask == 0, probabilities.shape)
```

Comment:
just a matter of taste, but I'd prefer 
```python
            probabilities *= (mask != 0)
```

instead of 

```python
            mask_all_channels = numpy.broadcast_to(mask == 0, probabilities.shape)
            probabilities[mask_all_channels] = 0.0
```
just because from the documentation ("Broadcast an array to a new shape.") it is not really clear to me what `broadcast_to` should to, how it fails, etc.

Reply:
I like your style better; Far more concise. It's probably doing almost the same behind the scenes, anyway. Since `mask` can have a shape that is different from that of `probabilities`, it still will have to be implicitly broadcast to match `probabilities`'s shape.

Your comment also made me realize that the previous code was misinterpreting the mask, making it work channel-wise. I've changed it to make the mask be  logical `OR` of all the channels, so that a user can have colored masks that will only cause the black pixels to be masked out